Makefile

```diff
@@ -144,3 +145,6 @@ stop-prod:
 .PHONY: environ-regen
 environ-regen:
 	bin/mk-environ-file.sh
+
+mjml:
```

Comment:
You may also want to add `front-deps` target as a dependency and extend it to install utility you need :)